### PR TITLE
Add gradient accumulation method for BYOL

### PR DESCRIPTION
**Pull request by scnuhealthy**
The linear evaluation result on the last layer on ImageNet val with the official tool is 67.29%. Batch size is 96(per gpu) * 7 (gpus) * 6 (interval) = 4032. Number of total epochs is 300, other hyperparameters are copied from the ablation study part of the BYOL paper.

### CODE REVIEW

**Comment by XiaohangZhan** [comment]
Sorry, we cannot merge it for now, see: https://github.com/open-mmlab/OpenSelfSup/issues/47#issuecomment-708438922
